tests/integration/features/overlay_partitions/lifecycle/test_chisel_lifecycle.py

```diff
@@ -16,5 +16,4 @@
 
 # Make pytest run any non-overridden chisel tests here with partitions enabled.
 
```

Comment:
Any file-wide rule-disables like this in tests can probably go into the `"tests/**.py"` section of ruff's per-file-ignores for the change. (In this case I'm thinking about F401 and F403)